apps/settings/src/components/PersonalInfo/BirthdaySection.vue

```diff
@@ -0,0 +1,141 @@
+<!--
+ - SPDX-FileCopyrightText: 2024 Nextcloud GmbH and Nextcloud contributors
+ - SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+
+<template>
+	<section>
+		<HeaderBar :scope="birthdate.scope"
+			:input-id="inputId"
+			:readable="birthdate.readable" />
+
+		<template>
+			<NcDateTimePickerNative :id="inputId"
+				type="date"
+				label=""
+				:value="value"
+				@input="onInput" />
+		</template>
+
+		<p class="property__helper-text-message">
+			{{ t('settings', 'Enter your date of birth') }}
+		</p>
+	</section>
+</template>
+
+<script>
+import HeaderBar from './shared/HeaderBar.vue'
+import AccountPropertySection from './shared/AccountPropertySection.vue'
+import { NAME_READABLE_ENUM } from '../../constants/AccountPropertyConstants.js'
+import { NcDateTimePickerNative } from '@nextcloud/vue'
+import debounce from 'debounce'
+import { savePrimaryAccountProperty } from '../../service/PersonalInfo/PersonalInfoService'
+import { handleError } from '../../utils/handlers'
+import AlertCircle from 'vue-material-design-icons/AlertCircleOutline.vue'
+import { loadState } from '@nextcloud/initial-state'
+
+const { birthdate } = loadState('settings', 'personalInfoParameters', {})
+
+export default {
+	name: 'BirthdaySection',
+
+	components: {
+		AlertCircle,
+		AccountPropertySection,
+		NcDateTimePickerNative,
+		HeaderBar,
+	},
+
+	data() {
+		let initialValue = null
+		if (birthdate.value) {
+			initialValue = new Date(birthdate.value)
+		}
+
+		return {
+			birthdate: {
+				...birthdate,
+				readable: NAME_READABLE_ENUM[birthdate.name],
+			},
+			initialValue: initialValue ? new Date(initialValue) : new Date(),
```

Comment:
```suggestion
			initialValue: initialValue ? initialValue : new Date(),
```

it already is a date object, right?

Reply:
Good catch! Done